src/app/components/Footer/__snapshots__/index.test.jsx.snap

```diff
@@ -0,0 +1,3 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Footer should render correctly 1`] = `<Footer />`;
```

Comment:
This is more of a comment on how Jest's snapshot works - this seems like an unhelpful snapshot since it doesn't include the actual HTML elements.  I'd have expected this snapshot file to include something like this:
```
<footer class="sc-bxivhb eoUnee"><p class="sc-ifAKCX kxMoYE">Copyright © 2018 BBC. The BBC is not responsible for the content of external sites.</p></footer>
```
It would be good to investigate why this snapshot currently just shows the React component `<Footer />` and not its constituent elements.